### PR TITLE
MODDATAIMP-309 - Data Import stops when trying to load a large file to folio-snapshot-load

### PR DESCRIPTION
**Pull request by RuslanLavrov**

### CODE REVIEW

File: mod-source-record-manager-server/src/test/java/org/folio/services/JobExecutionProgressServiceImplTest.java [hunk 1]
```diff
@@ -102,14 +104,15 @@ public void shouldNotReturnFailedFutureWhenJobExecutionIdDuplicates(TestContext
     Async async = context.async();
     int expectedTotalRecords = 62;
 
-    Future<JobExecutionProgress> future = jobExecutionService.initializeJobExecutions(initJobExecutionsRqDto, params)
-      .compose(initJobExecutionsRsDto -> jobExecutionProgressService.initializeJobExecutionProgress(initJobExecutionsRsDto.getParentJobExecutionId(), expectedTotalRecords, TENANT_ID)
-        .compose(jobExecutionProgress -> jobExecutionProgressService.initializeJobExecutionProgress(jobExecutionProgress.getJobExecutionId(), expectedTotalRecords, TENANT_ID)));
+    Future<CompositeFuture> future = jobExecutionService.initializeJobExecutions(initJobExecutionsRqDto, params)
+      .compose(initJobExecutionsRsDto -> {
+        Future<JobExecutionProgress> future1 = jobExecutionProgressService.initializeJobExecutionProgress(initJobExecutionsRsDto.getParentJobExecutionId(), expectedTotalRecords, TENANT_ID);
+        Future<JobExecutionProgress> future2 = jobExecutionProgressService.initializeJobExecutionProgress(initJobExecutionsRsDto.getParentJobExecutionId(), expectedTotalRecords, TENANT_ID);
```
Comment (KaterynaSenchenko) [inline mod-source-record-manager-server/src/test/java/org/folio/services/JobExecutionProgressServiceImplTest.java:110]: aren't they identical?

Comment (KaterynaSenchenko) [inline mod-source-record-manager-server/src/test/java/org/folio/services/JobExecutionProgressServiceImplTest.java:110]:
```suggestion
        Future<JobExecutionProgress> future2 = jobExecutionProgressService.initializeJobExecutionProgress(initJobExecutionsRsDto.getJobExecutionId(), expectedTotalRecords, TENANT_ID);
```

Comment (RuslanLavrov) [reply mod-source-record-manager-server/src/test/java/org/folio/services/JobExecutionProgressServiceImplTest.java:110]: yes, they are identical, two concurrent attempts to create progress with the same jobExecution id

Comment (RuslanLavrov) [reply mod-source-record-manager-server/src/test/java/org/folio/services/JobExecutionProgressServiceImplTest.java:110]: initJobExecutionsRsDto doesn’t have getJobExecutionId() method

Comment (KaterynaSenchenko) [reply mod-source-record-manager-server/src/test/java/org/folio/services/JobExecutionProgressServiceImplTest.java:110]: oops, you're right, missed that it's a test

**Pull request merged**